# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../lualdap/config-no-compat-5.1.patch
# Copyright (C) 2006 - 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./config.orig	2007-12-14 18:06:58.000000000 +0100
+++ ./config	2012-05-24 10:31:06.564841080 +0200
@@ -1,8 +1,8 @@
 # Installation directories
 # System's libraries directory (where binary libraries are installed)
-LUA_LIBDIR= /usr/local/lib/lua/5.0
+LUA_LIBDIR= $(shell pkg-config --variable INSTALL_CMOD lua)
 # Lua includes directory
-LUA_INC= /usr/local/include
+LUA_INC= $(shell pkg-config --variable includedir lua)
 # OpenLDAP includes directory
 OPENLDAP_INC= /usr/local/include
 # OpenLDAP library (an optional directory can be specified with -L<dir>)
@@ -13,13 +13,12 @@
 #LIB_OPTION= -bundle -undefined dynamic_lookup #for MacOS X
 
 # Lua version number (first and second digits of target version)
-LUA_VERSION_NUM= 500
+LUA_VERSION_NUM= $(shell pkg-config --variable=V lua | tr '.' '0')
 LIBNAME= $T.so.$V
-COMPAT_DIR= ../compat/src
 
 # Compilation parameters
 WARN= -O2 -Wall -fPIC -W -Waggregate-return -Wcast-align -Wmissing-prototypes -Wnested-externs -Wshadow -Wwrite-strings -ansi
-INCS= -I$(LUA_INC) -I$(OPENLDAP_INC) -I$(COMPAT_DIR)
+INCS= -I$(LUA_INC) -I$(OPENLDAP_INC)
 CFLAGS= $(WARN) $(INCS)
 CC= gcc
 
